test/components/connectors/test_openapi_service.py

```diff
@@ -1,15 +1,23 @@
 import json
 import os
 import time
-from unittest.mock import MagicMock, Mock
+from typing import Dict, Any
+from unittest.mock import MagicMock, patch
+from urllib.parse import urlparse
 
 import flaky
 import pytest
 import requests
+from flask import Flask, jsonify, request
+from flask.testing import FlaskClient
```

Comment:
I agree, there's no need to spawn a local server to test this Component. 
Let's just go with unit tests that mock the OpenAPI library, otherwise we're not testing our code but OpenAPI's.

Another downside is that it complicates tests quite a bit.